### ISSUE TITLE
Can we edit multiple overleaf files in this?

### ISSUE THREAD

**Issue by saurabh-kataria** [fjwillemsen/NativeOverleaf#9]
If someone is working on multiple overleaf projects, can they use your app?

**Comment by fjwillemsen** [comment]
Yes, opening multiple instances of the app is enabled, but it depends on the operating system how easy it is. For macOS, you have to do `open -n -a Overleaf` in the Terminal (there is no native GUI way to do this). For Windows, you can shift + click on the icon. For Linux, it depends on your distribution. I would need to check how this affects local storage though, which means that data being saved in local storage such as the word count can potentially become inconsistent with multiple instances open at once. I will get back on this.

**Comment by ysfkrl** [comment]
Actually in macos I just right click the home button and select "open in new window" or command + click to open in new tab then drag the tab outside to create a new window. 

<img width="332" alt="image" src="https://user-images.githubusercontent.com/89781812/228674660-6bd4ce9a-99c2-4b5f-b2ab-41ed9f0da422.png">
